### PR TITLE
KOGITO-4783 - SW and Funqy Example

### PR DESCRIPTION
**Pull request by tsurdilo**
Signed-off-by: Tihomir Surdilovic <tsurdilo@redhat.com>

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [ ] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains link to any dependent or related Pull Request
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>
</details>

### CODE REVIEW

File: serverless-workflow-funqy/sw-funqy-services/pom.xml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-health</artifactId>
+        </dependency>
+
+        <!-- Serverless Workflow -->
+        <dependency>
+            <groupId>org.kie.kogito</groupId>
+            <artifactId>kogito-openapi-workitem</artifactId>
```
Comment (ricardozanini) [inline serverless-workflow-funqy/sw-funqy-services/pom.xml]: I think we don't need this dependency :thinking: 

Is this the funqy module, correct?

File: serverless-workflow-funqy/sw-funqy-services/pom.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            <artifactId>quarkus-jsonp</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest-client</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
+        </dependency>
```
Comment (ricardozanini) [inline serverless-workflow-funqy/sw-funqy-services/pom.xml]: Same here, I don't think we need these dependencies

File: serverless-workflow-funqy/sw-funqy-workflow/pom.xml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            <groupId>io.rest-assured</groupId>
+            <artifactId>rest-assured</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-smallrye-health</artifactId>
+        </dependency>
+
+        <!-- Serverless Workflow -->
```
Comment (ricardozanini) [inline serverless-workflow-funqy/sw-funqy-workflow/pom.xml:68]: Once this gets merged: https://github.com/kiegroup/kogito-runtimes/pull/1153, we can cut all of these deps.

**Review by ricardozanini — COMMENTED** [review]
Pretty cool! Can you please just review the dependencies in the `*-services` module?

**Comment by tsurdilo** [comment]
@ricardozanini ok i will review all the depends and remove ones not needed

**Comment by tsurdilo** [comment]
@ricardozanini updated poms

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
Successfully deployed on Kubernetes with Knative. :+1:

**Review by cristianonicolai — APPROVED** [review]
nice work @tsurdilo

**Pull request merged**